designs/administrator/olm/operand-creation-at-operator-install/index.md

```diff
@@ -0,0 +1,36 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Allow Operand (Custom Resource) to be Created During Operator Installation
+
+Currently some operators need the user to instantiate a custom resource for the operator to be fully functional, but it is challenging for the user to determine they need to do so today. This design allows the operator developer to specify a single required custom resource that should be created at the same time as the operator is installed.
+
+## Installing an Operator with a Required Operand Specified
+
+![Install operator](img/2-1-install-needCR.png)
+- The user selects the operator they'd like to install and selected installation options.
+- The required custom resource is marked with a badge as **Required**.
+
+![Installing operator](img/2-2-installing.png)
+- The ‘Installing’ screen appears, though the required custom resource is now shown with the **Create [Custom Resource]** button not yet enabled.
+- The user can navigate away from this screen, but they may be more inclined to stay seeing that the custom resource creation is next.
+
+![Installed create CR](img/2-3-instantiateCR.png)
+- Once the operator has successfully installed, the user is notified that they still need to create the required custom resource to use the operator.
+
+![Create custom resource](img/2-4-createCR.png)
+- Existing interactions are used to create the required custom resource.
```

Comment:
It could be worth mentioning:
_if users hit any errors on this createCR view (e.g. _namespaces "openshift-storage" not found_ for OCS Operator), users could safely navigate away to address the issue first, and later come back to previous "instantiateCR" (the one with CTA button to create CR) from the "Installed Operators" list view instead of being sent to the regular CSV Details view._ 

This way we make sure users still being helped to know which CR they need to create if they ran into issues during the CR creation process.

Reply:
In that case I guess I was expecting the user to end up in the CSV Details with the warning inline alert & create CR CTA that is conveyed in the last screen of this design. Would we otherwise need to change the logic of when the installing screen appears vs not?

Reply:
You're right, the last screen of this design doc should cover this case pretty well. 👍 👍